packages/web3-eth-accounts/src/account.ts

```diff
@@ -0,0 +1,70 @@
+import { utils, getPublicKey } from 'ethereum-cryptography/secp256k1';
+import { toChecksumAddress, bytesToHex, sha3Raw, HexString } from 'web3-utils';
+import { InvalidPrivateKeyError, PrivateKeyLengthError } from './errors';
+
+// Will be added later
+export const encrypt = (): boolean => true;
+
+// Will be added later
+export const sign = (): boolean => true;
+
+// Will be added later
+export const signTransaction = (): boolean => true;
```

Comment:
```suggestion
// TODO Will be added later
export const encrypt = (): boolean => true;

// TODO Will be added later
export const sign = (): boolean => true;

// TODO Will be added later
export const signTransaction = (): boolean => true;
```

Adding `TODO` makes it easier to find using an IDE

Reply:
Or this might not be needed if it's tracked in an issue. I just like the idea of all TODOs being labeled in the code so we can search for anything we might've missed